examples/X11/X11.cpp

```diff
@@ -93,7 +93,7 @@
     glRotatef(elapsedTime * 18.f, 0.f, 0.f, 1.f);
 
     // Define a 3D cube (6 faces made of 2 triangles composed by 3 vertices)
-    static const GLfloat cube[] =
+    constexpr std::array<GLfloat, 216> cube =
```

Comment:
I guess we are not using CTAD here because we need to make sure that the values are `GLfloat`, right?

Reply:
That's correct.